l10n_es_aeat_sii_oss/tests/test_l10n_es_aeat_sii.py

```diff
@@ -35,6 +35,7 @@ def setUpClass(cls):
 
     def test_invoice_sii_oss(self):
         self.partner.sii_simplified_invoice = True
+        self.env.clear()
```

Comment:
No tengo claro por qué necesitáis hacer esto. Puede estar tapando un problema en otro lado.

Reply:
Aquí @pedrobaeza  estuvimos viendo que si no borrábamos la cache estaba intentando crear X registros nuevos (NewID) y nos saltaba error al hacer el sorted del metodo _compute_name.

https://github.com/OCA/OCB/blob/289e62d730435f7c9017e7f5f08788d6938ad5cb/addons/account/models/account_move.py#L683

Es posible que haya un problema con el Form y que no este borrando la cache o que este manteniendo registros en cache que no tendría que darse, pero no hemos dado con ello.

Sabes si a alguien le ha pasado esto en los test con el Form, ya que también cambiamos el Form por un create del registro y el test paso bien.

Un saludo

Reply:
Ahora las facturas ya no se deberían crear con `Form` en v16. Se volvió a tocar para que con los mínimos `vals`, fuera creable desde código. Echa un vistazo a este método y si acaso, cambiad el código para utilizarlo de forma similar:

https://github.com/odoo/odoo/blob/f73c214efdbd49865ff3f0db915fadc80b4403b0/addons/account/tests/common.py#L674

Reply:
Buenos días.

He modificado el test para hacerlo con `create` directamente. El test no era hijo de `AccountTestInvoicingCommon` por eso al final he hecho el `create` directamente. El número de líneas es mayor que cuando se hacía con `Form` pero no es nada significativo.